packages/just-scripts/src/tasks/jestTask.ts

```diff
@@ -24,16 +29,17 @@ export function jestTask(options: JestTaskOptions = {}): TaskFunction {
       const cmd = process.execPath;
 
       const args = [
+        ...(options.nodeArgs || []),
         jestCmd,
         '--config',
         configFile,
         '--passWithNoTests',
         '--colors',
-        options.runInBand ? '--runInBand' : '',
-        options.coverage ? '--coverage' : '',
-        options.watch ? '--watch' : '',
-        options.u || options.updateSnapshot ? '--updateSnapshot' : '',
-        ...(options._ ? options._ : [])
+        ...(options.runInBand ? ['--runInBand'] : []),
```

Comment:
wha'ts with the change to array/spread here?

Reply:
change here means that the resultant array would not include those blanks...